composer.json

```diff
@@ -45,10 +49,10 @@
         "doctrine/dbal": "^3.1",
         "geocoder-php/cache-provider": "^4.3",
         "geocoder-php/nominatim-provider": "^5.5",
-        "laragear/webauthn": "1.x-dev",
-        "laravel/framework": "^9.2",
-        "lychee-org/nestedset": "^6",
-        "lychee-org/php-exif": "^0.8",
+        "laragear/webauthn": "laravel-10-dev",
```

Comment:
Yeah I really appreciate that this maintainer pretty much use my PR as template and just rewrite everything...
I still need to investigate why some parts of U2F are not working. I suspect it is because it is limited to a subset of cryptographic key types (namely RSA and no ECDSA).

Reply:
@qwerty287 Fixed in [1b99e6a](https://github.com/LycheeOrg/Lychee/pull/1764/commits/1b99e6ac6b1553dfd80db47526d98f364f7110cf)